liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -666,10 +666,10 @@ protected void doExecute() throws Exception {
         final ThreadPoolExecutor executor = new ThreadPoolExecutor(1, 1, 0L, TimeUnit.MILLISECONDS,
                 new ArrayBlockingQueue<Runnable>(1, true));
 
-        runMojo("org.apache.maven.plugins", "maven-compiler-plugin", "compile");
-        runMojo("org.apache.maven.plugins", "maven-resources-plugin", "resources");
-        runMojo("org.apache.maven.plugins", "maven-compiler-plugin", "testCompile");
-        runMojo("org.apache.maven.plugins", "maven-resources-plugin", "testResources");
+        runCompileMojo();
+        runResourcesMojo();
```

Comment:
With the latest changes, the runResourcesMojo and runTestResourcesMojo methods are somewhat unnecessary since they are just being called once.